package.json

```diff
@@ -1,18 +1,20 @@
 {
   "name": "egghead-ui",
-  "version": "4.7.2",
+  "version": "5.0.0",
```

Comment:
Major version bump; I didn't consciously make any breaking changes here, but consumers should update knowing breaking changes are possible with such a large refactor